### PR TITLE
doc: Release notes v1.3

### PR DESCRIPTION
**Pull request by lirui34**
Release notes v1.3

Signed-off-by: lirui34 <ruix.li@intel.com>

### CODE REVIEW

File: doc/release_notes/release_notes_1.3.rst [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+* Platform-level coordinated graceful shutdown (S5) for User VMs.
+* Virtual UART (vUART) for inter-VM communication.
+* Introduced :ref:`acrn-config <acrn_configuration_tool>` tool to configure VM
+  and hypervisor from a XML configuration file at build time.
+* Ethernet mediator now supports prioritization per VM.
+* Features for real-time determinism, e.g. Cache Allocation Technology (CAT, only supported on Apollo Lake).
+
+Document updates
+================
+We have many `reference documents available <https://projectacrn.github.io>`_, including:
```
Comment (gvancuts) [inline doc/release_notes/release_notes_1.3.rst]: How about this minor edit?
```suggestion
We have many new `reference documents available <https://projectacrn.github.io>`_, including:
```

File: doc/release_notes/release_notes_1.3.rst [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+* Getting Start Guide for Industry scenario
```
Comment (gvancuts) [inline doc/release_notes/release_notes_1.3.rst]:
```suggestion
* Getting Started Guide for Industry scenario
```

Could we not already add the link, i.e. **:ref:`<rt_industry_setup>`**

File: doc/release_notes/release_notes_1.3.rst (hunk 1 above)
Comment (fuzhongl) [reply doc/release_notes/release_notes_1.3.rst]: Done!

File: doc/release_notes/release_notes_1.3.rst (hunk 2 above)
Comment (fuzhongl) [reply doc/release_notes/release_notes_1.3.rst]: Done!

**Comment by wenlingz** [comment]
Have update release_1.3 related version info, please help merge

**Pull request merged**